src/modules/dashboard/components/Members/MembersSection.tsx

```diff
@@ -0,0 +1,100 @@
+import React, { useState, useCallback } from 'react';
+
+import { FormattedMessage, MessageDescriptor } from 'react-intl';
+import { ColonyRole } from '@colony/colony-js';
+import styles from './MembersSection.css';
+
+import MembersList from '~core/MembersList';
+import { AnyUser, Colony, useMembersSubscription } from '~data/index';
+
+import UserPermissions from '~dashboard/UserPermissions';
+import LoadMoreButton from '~core/LoadMoreButton';
+
+const displayName = 'dashboard.MembersSection';
+
+type Member = AnyUser & {
+  roles: ColonyRole[];
+  directRoles: ColonyRole[];
+  banned: boolean;
+};
+
+interface Props {
+  title: MessageDescriptor;
+  description?: MessageDescriptor;
+  sortingParams?: string[];
+  colony: Colony;
+  currentDomainId: number;
+  members: Member[];
+}
+
+const ITEMS_PER_SECTION = 10;
+const MembersSection = ({
+  title,
+  description,
+  colony: { colonyAddress },
+  colony,
+  currentDomainId,
+  members,
+}: // @NOTE Add another optional paramater called sortingOptions to handle sorting
+Props) => {
+  const [dataPage, setDataPage] = useState<number>(1);
+
+  /*
+   * @NOTE Poor man's pagination
+   *
+   * We're not really doing proper pagination like we do for say... `ColonyEvents`,
+   * because our metadata server doesn't support skip, first, more, next, etc...
+   * query filters
+   *
+   * A proper solution for this would be to teach the server to return just part
+   * of the results, with the next waiting in line for a callback, but the current
+   * timeframe doesn't allow for this, so I guess this is the "best" we can do for now
+   */
+
+  const paginatedMembers = members.slice(0, ITEMS_PER_SECTION * dataPage);
+  const handleDataPagination = useCallback(() => {
+    setDataPage(dataPage + 1);
+  }, [dataPage]);
```

Comment:
So I'm not really sure but I think pagination specs look different for this page now.

Please check with Product to figure out what these are.